td-agent/yum/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -exu
+
+# Amazon Linux 2 system-release-cpe is:
+# cpe:2.3:o:amazon:amazon_linux:2
+# CentOS 7 system-release-cpe is:
+# cpe:/o:centos:centos:7
+# CentOS 6 system-release-cpe is:
+# cpe:/o:centos:linux:6
+# This means that column glitch exists.
+# So, we should remove before "o" character.
+
+distribution=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f2)
+version=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f4)
+
```

Comment:
Another solution:

`$(cat /etc/system-release-cpe | sed -e 's/.\+o:\([a-z]\+\?\):.\+/\1/')`

and

`$(cat /etc/system-release-cpe | sed -e 's/.\+:\([0-9]\+\)$/\1/')`

Reply:
I feel that using awk is easier for me.

Reply:
okay.